src/baseDesign/RootStyle.tsx

```diff
@@ -33,4 +37,10 @@ export const RootStyle = createGlobalStyle`
       --easing-decelerate: cubic-bezier(0.0, 0.0, 0.2, 1);
       --easing-accelerate: cubic-bezier(0.4, 0.0, 1, 1);
     }
+
+    @font-face {
+      font-family: "Inter";
+      src: url("/fonts/Inter-VariableFont_slnt,wght.ttf");
+      font-weight: 100 900;
```

Comment:
Ajoute ça pour éviter que tu ais du clipping de font
```suggestion
      font-weight: 100 900;
      font-display: block;
```

Reply:
done